resources/deprecations.properties

```diff
@@ -150,6 +159,7 @@ poll-mailbox-trigger = https://git.io/JfaQM
 pretest-commit = https://wiki.jenkins.io/x/5gB-B
 # https://wiki.jenkins.io/display/JENKINS/pucm+plugin
 PUCM = https://wiki.jenkins.io/x/fIBVAw
+rad-builder = https://groups.google.com/g/jenkinsci-dev/c/9M2B6ZTKjI0/m/BChinuUVBAAJ
```

Comment:
```suggestion
```

The plugin still works (I assume -- at least the core PR didn't change anything about that). It can be installed from update sites and it will pull all needed dependencies. This only affects very narrow use cases around out-of-process plugin management.

> <img width="593" alt="Screenshot 2021-05-26 at 10 27 35" src="https://user-images.githubusercontent.com/1831569/119628139-1caec480-be0d-11eb-9837-bf0ba634d75d.png">